test/moq_messages.cpp

```diff
@@ -552,7 +552,12 @@ ObjectDatagramEncodeDecode(bool extensions)
     object_datagram.object_id = 0xFF;
     object_datagram.priority = 0xA;
     object_datagram.extensions = extensions ? kOptionalExtensions : std::nullopt;
-    object_datagram.payload = { 0x1, 0x2, 0x3, 0x5, 0x6 };
+    if (empty_payload) {
+        object_datagram.object_status = quicr::ObjectStatus::DOES_NOT_EXIST;
+        object_datagram.payload = {};
+    } else {
+        object_datagram.payload = { 0x1, 0x2, 0x3, 0x5, 0x6 };
+    }
```

Comment:
Something about manually setting this status and then having to even care about the payload contents seems odd... Shouldn't it be one or the other? If the status is DOES_NOT_EXIST then who cares what's in the payload?